test/files/neg/named-booleans-relaxed.check

```diff
@@ -0,0 +1,21 @@
+named-booleans-relaxed.scala:22: warning: Boolean literals should be passed using named argument syntax for parameter x.
+  val x0 = c.f(17, true, false) // warn
```

Comment:
Add a case for a single `boolean` parameter, which expected no warning.

Reply:
`c.check(true, "OK")` but it also does not warn in "strict" mode because of Lukas rules.